src/Psalm/Type/Atomic/TCallableObject.php

```diff
@@ -26,7 +26,7 @@ public function toPhpString(
         ?string $this_class,
         int $analysis_php_version_id
     ): ?string {
-        return $analysis_php_version_id >= 72000 ? 'object' : null;
```

Comment:
This is actually a bug, there's no PHP 7.20, and there never will be.

Reply:
oh nice